microservices/product-search/README.md

```diff
@@ -0,0 +1,97 @@
+# Microservice Product Search
+
+Microservice responsible for product search via Elasticsearch on [Shopsys Framework](https://www.shopsys-framework.com).
+
+This repository is maintained by [shopsys/shopsys](https://github.com/shopsys/shopsys) monorepo, information about the changes are in [monorepo CHANGELOG.md](https://github.com/shopsys/shopsys/blob/master/CHANGELOG.md).
+
+## How it works
+Microservice acts as a fully independent unit.
+It uses its own separate server, separate logic and it uses own vendor directory for Composer dependencies.
+All of these parts are located in a separate Docker container.
+This container is configured in [docker-compose.yml](https://github.com/shopsys/shopsys/tree/master/docker/conf) under the name `microservice-product-search`
+
+As the base of the microservice, we used micro-kernel which is made up of [Symfony 4](https://symfony.com/4).
+This leanest Symfony version is highly optimized and it is suitable for this type of project.
+
+For the search process is used a super fast no-SQL database Elasticsearch. For more information about the Elasticsearch on Shopsys Framework, see [Product search via Elasticsearch]((https://github.com/shopsys/shopsys/blob/master/docs/introduction/product-search-via-elasticsearch.md).) 
```

Comment:
There seems to be misplaced parenthesis in the end -> the link doesn't work

Reply:
This status is correct. README.md is in a different repo. So you can not use a relative path. This link will be functional after merging into master. You can use link into the current branch, but after merge into master and after removal of this old branch, your link will be broken.